steveify1/capcoinfx-api
# #12 chore(badges):add badges to README.md

#### What does this PR do?
add badges to README.md
#### Description of Task to be completed?
Add the following badges to README.md:
- travis
- coveralls
- codeclimate
#### How should this be manually tested?
- clone the repository and cd into the project directory
- check the `README.md` file for updates
- visit the homepage of this repo to find badges
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?
#12 
#### Screenshots (if appropriate)
#### Questions:
